### PR TITLE
Cardview refactoring

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Was experimenting with design changes in the card but I didn't like them. Did some refactoring while I was at it though.

Design/behavior is exactly the same as before.

**Changes**
- Format XML/Java code
- Switch to view binding
- Remove unused branches
- Make it slightly easier to work with the Android Studio designer

**Issues**

### CODE REVIEW

**Review by thornbill — APPROVED** [review]

**Comment by linetrimmer** [comment]
On series with unwatched episodes, I'm getting the unwatched count and the check mark instead of the unwatched count and the blue circle. Seems like the circle switched from 'watchedIndicator' to 'watched' and the check mark went from 'watched' to 'watchedIndicator'

Also items in thumbnail or banner view have a folder icon in the bottom left corner of the card

**Comment by nielsvanvelzen** [comment]
Thanks, will look into that later

**Comment by linetrimmer** [comment]
#747 for the check mark/blue circle issue

File: app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyImageCardView.java [hunk 1]
```diff
@@ -1,120 +1,59 @@
...
-        mImageView = (ImageView) v.findViewById(R.id.main_image);
-        mInfoArea = v.findViewById(R.id.info_field);
-        mTitleView = (TextView) v.findViewById(R.id.title_text);
-        mContentView = (TextView) v.findViewById(R.id.content_text);
-        mBadgeImage = (ImageView) v.findViewById(R.id.extra_badge);
-        mOverlayName = (TextView) v.findViewById(R.id.overlay_text);
-        mOverlayCount = (TextView) v.findViewById(R.id.overlay_count);
-        mOverlayIcon = (ImageView) v.findViewById(R.id.icon);
-        mInfoOverlay = (ViewGroup) v.findViewById(R.id.name_overlay);
-        mInfoOverlay.setVisibility(GONE);
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyImageCardView.java:83]: ^ This line should've been kept, making a PR now

**Pull request merged**